trieste/acquisition/function/function.py

```diff
@@ -223,12 +232,132 @@ def __call__(self, x: TensorType) -> TensorType:
         return (self._eta - mean) * normal.cdf(self._eta) + variance * normal.prob(self._eta)
 
 
+class log_expected_improvement(AcquisitionFunctionClass):
```

Comment:
you can inherit from `expected_improvement`, no?
then there is no need for init and update methods...

Reply:
docstring can be moved to class docstring

Reply:
Fixed in https://github.com/secondmind-labs/trieste/pull/904/commits/2f45eb8e8035fbfa7105f6f2acd9a470edb32906